plugin/src/ConCat/Plugin.hs

```diff
@@ -454,6 +454,10 @@ ccc (CccEnv {..}) (Ops {..}) cat =
         sub = [(x,mkEx funCat exlV (Var z)),(y,mkEx funCat exrV (Var z))]
         -- TODO: consider using fst & snd instead of exl and exr here
         mbe' = mkCurry' cat (mkCcc (Lam z (subst sub e)))
+     Trying("lam boxer")
+     (boxCon -> Just e') ->
+       Doing("lam boxer")
+       return (mkCcc (Lam x e'))
```

Comment:
Do you remember how you came to add this case?

Reply:
Comparing the paper with the plugin, I determined that something like this was missing - the rules couldn't possibly have worked without this. Then I dug back in the git history to the point where the old rules still were, and there it was.

Reply:
Got it. Thanks.